Data/Sys/GameSettings/RODE01.ini

```diff
@@ -25,6 +25,5 @@ PH_ExtraParam = 0
 PH_ZNear =
 PH_ZFar =
 
-[Video_Hacks]
-EFBEmulateFormatChanges = True
-
+[Video_Settings]
+AspectRatio = 2
```

Comment:
I don't think enforcing a certain aspect ratio is a good idea whatever the reason. It is a matter of individual preference what is better and that is hindered this way. Also why remove emulate format changes? Check this thread on why it was put:
https://forums.dolphin-emu.org/Thread-wii-wario-ware-smooth-moves-pal